src/streams.h

```diff
@@ -530,6 +530,14 @@ class AutoFile
     //
     // Stream subset
     //
+    void seek(int64_t offset, int origin)
+    {
+        if (!file)
```

Comment:
1d093bbf97b8d4ec5413ca560ac79e56f23d7f1d `{`

Could also use the `IsNull()` helper.

Reply:
Done